src/data/networks/index.js

```diff
@@ -0,0 +1,24 @@
+import { arbitrumMainnet, arbitrumTestnet } from './arbitrum'
+import { polygonMainnet, polygonTestnet }   from './polygon'
```

Comment:
```suggestion
import { polygonMainnet, polygonTestnet } from './polygon'
```
Just because it is like this throughout the rest of the app.